django_project/geosight/data/serializer/dashboard.py

```diff
@@ -315,7 +315,7 @@ def get_permission(self, obj: Dashboard):
     class Meta:  # noqa: D106
         model = Dashboard
         fields = (
-                     'id', 'slug', 'icon', 'name',
+                     'id', 'slug', 'icon', 'thumbnail', 'name',
```

Comment:
thumbnail here can just contains function tha
check if icons with /icon/thumbnail exist, return it
if not exist, return icon url